File: Exemplos/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Java-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/TERMICA/Corte.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA

import com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.TermicaCommand

class Corte(private val avanco: Int) : TermicaCommand("Corte") {
    override fun functionParameters(): String {
        return "\"avanco\"" + ":" + avanco
    }
}

File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/TERMICA/corte.dart
import 'package:intent_digital_hub/IntentDigitalHubService/TERMICA/termica_commad.dart';

class Corte extends TermicaCommand {
  final int _avanco;

  Corte(this._avanco) : super('Corte');

  @override
  functionParameters() => '"avanco"' ':' '$_avanco';
}
